NEWS

```diff
@@ -11,6 +11,9 @@ CHANGES WITH 235:
           that case users will be prevented from correctly managing bond0
           interface using networkd.
 
+        * clock file is now moved to /var/lib/systemd/timesync/clock.
+          Distributors should change the location of the file if it is packaged.
+
```

Comment:
I don't think this file is packaged with systemd. The minor issue here would happen during a restart after an upgrade as the old timesyncd would use the old path and the new timsyncd the new path thus loosing the recorded timestamp. This will also leave an used file in place.

I would suggest:
The file /var/lib/systemd/clock used by systemd-timesyncd to remember the last known good clock value has been moved to /var/lib/systemd/timesync/clock. Distributors should either remove the previous file after the service has been restarted or move the current file to the new directory once the old daemon has been stopped and before the new daemon startup.

But this may be to much of a hassle for such a minor issue / minor gain.